poetry.lock

```diff
@@ -0,0 +1,2070 @@
+# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
```

Comment:
previously we had no poetry lock and just assumed the builds would use the newest versions of everything. Intention being we catch things as early as possible if libs get updated. Installs of sequence metrics will not necessarily use these locked versions.

Reply:
Got it, will remove.